LEAF_Request_Portal/admin/templates/mod_form.tpl

```diff
@@ -58,6 +68,19 @@ function openContent(url) {
                                  <td><input id="sort" type="number"></input></td>\
                              </tr>\
                            </table>');
+        $.ajax({
+            type: 'GET',
+            url: '../api/form/_' + currCategoryID,
+            success: function(res) {
+                if(res.length > 0) {
+                    $("#needToKnow").val(checkSensitive(res));
+                    if($("#needToKnow").val() === '1') {
+                        $("#needToKnow option[value='0']").remove();
+                        $("#needToKnow option[value='1']").html('Forced on because sensitive fields are present');
```

Comment:
Set the value of `#needToKnow` within the conditional here. If the form is set to need to know in the `Edit Properties` popup, but has no sensitive indicators, it will show as `off` instead of `on`. Just use the call to  `checkSensitive()` in the conditional: `if(checkSensitive(res) === 1) `

Reply:
Addressed in commit `7a4`